### ISSUE TITLE
Minor typo in Readme.md?

### ISSUE THREAD

**Issue by vikram-s-narayan** [JuliaGaussianProcesses/AbstractGPs.jl#289]
In the Readme.md example: 

```
# Generate toy synthetic data.
x = rand(10)
y = sin.(rand(10))
```
should be:

```

# Generate toy synthetic data.
x = rand(10)
y = sin.(x)
```

**Comment by willtebbutt** [comment]
Good catch. Could you open a PR to make the fix?

**Comment by st--** [comment]
Closed by #291
